swagger/swagger.yaml

```diff
@@ -739,6 +744,39 @@ components:
           $ref: "#/components/schemas/ThreatTypeEnum"
         severity:
           $ref: "#/components/schemas/ThreatSeverityEnum"
+    Event:
+      required:
+        - unstructured
+        - type
+      type: object
+      description: an unit of structured communication from scenario to ADM
```

Comment:
This is so nitpicky, but it should be "a unit" instead of an (Sorry don't kill me):

For words where the “u” sound is pronounced like “you,” such as user, usual, or utilized, use “a.”
For words where the “u” sound is pronounced like “uh,” such as unusual, understanding, or utter, use “an.”

Reply:
Not too nitpicky!  I agree and I'll fix it!  (Thought I might fix it in ITM-472 depending how many other issues there are.)

Reply:
This is addressed in ITM-477's [PR](https://github.com/NextCenturyCorporation/itm-evaluation-server/pull/77).